qa/test-db-instance-migration/test-fixture-719/pom.xml

```diff
@@ -48,6 +48,17 @@
       <artifactId>camunda-qa-upgrade-scenario-util</artifactId>
       <version>${project.version}</version>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.openjpa</groupId>
+      <artifactId>openjpa</artifactId>
+      <scope>test</scope>
```

Comment:
❓ Any specific reason that this is only in `test` scope while the others are not?

Reply:
Non, I will remove the scope.